packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -16,7 +16,9 @@ export const samplerOperations = {
                     .getABIEncodedTransactionData();
             },
             handleCallResultsAsync: async (contract, callResults) => {
-                return contract.getABIDecodedReturnData<BigNumber[]>('getOrderFillableTakerAssetAmounts', callResults);
+                return contract
+                    .getABIDecodedReturnData<BigNumber[]>('getOrderFillableTakerAssetAmounts', callResults)
```

Comment:
I'm assuming they're not signing the orders where the intent to fill is false? Rather than use `takerAddress` as a conditional can we determine this off the signature, or lack there of?

Reply:
I'm sorry, I wrote this commit while blindly following the spec I wrote.  In retrospect, this idea, of making the sampler bypass fillability checks because these orders are unsigned, is moot now.

This is because we're getting indicative quotes in parallel with the on-chain call, not prior to it (like we do for firm quotes), and we're piping the quotes directly into the order router, and there's no further fillability check after routing*, so there's no need for such a hack.

It was only ever consideration when we were trying to decide whether to request quotes from `SwapQuoter` or from `MarketOperationUtils`, but ultimately we decided to do both (for latency reasons), alleviating the need for this.

*: Of course there's the gas estimation / transaction validation stage that happens in the 0x-api side, but you'll see [there](https://github.com/0xProject/0x-api/commit/66c9d9dd3aa71b96b2a3b85000f9c0a44bc0bffd#diff-aa8361c6f3b8881b3948134ce5805f8eR53) that I'm forcing `skipValidation` to true when handling `/swap/price` requests and delegating them to the handler for `/swap/quote`.